test/test_runner_battle.c

```diff
@@ -866,6 +959,7 @@ static const char *const sEventTypeMacros[] =
     [QUEUED_ABILITY_POPUP_EVENT] = "ABILITY_POPUP",
     [QUEUED_ANIMATION_EVENT] = "ANIMATION",
     [QUEUED_HP_EVENT] = "HP_BAR",
+    [QUEUED_EXP_EVENT] = "EXP_BAR",
```

Comment:
```suggestion
    [QUEUED_EXP_EVENT] = "EXPERIENCE_BAR",
```
These use the names as they appear in the source code so that the error message can be like "file.c:84: EXPERIENCE_BAR didn't match" and when you look at line 84 you can see `EXPERIENCE_BAR`.